src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -71,6 +71,7 @@ struct ConcatListOfFrames
              ++linearBlockIdx
              )
         {
+            T_Filter filter = filterWindowDomain;
```

Comment:
yes, please.

and you are sure you can't/don't need to add more instructions in the `pragma omp parallel for` block?